pkg/cloud/cloud_suite_test.go

```diff
@@ -17,13 +17,76 @@ limitations under the License.
 package cloud_test
 
 import (
+	"os"
+	"strings"
 	"testing"
 
+	"github.com/apache/cloudstack-go/v2/cloudstack"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"sigs.k8s.io/cluster-api-provider-cloudstack/pkg/cloud"
+	"sigs.k8s.io/cluster-api-provider-cloudstack/test/dummies"
+	"sigs.k8s.io/cluster-api-provider-cloudstack/test/helpers"
+)
+
+var (
+	realCloudClient cloud.Client
+	client          cloud.Client
```

Comment:
could you clarify the difference between these two? either a comment or renaming the variable. I'm not sure what `realCloudClient` means here either, that could be referring to anything in this day and age. Maybe mention that one is for the temporary test account user, and the other is for the credentials from the config file

Reply:
`cloud` is the cloud package being tested.

realCloudClient is a cloud.Client with a true connection to Apache CloudStack

Otherwise during testing we mock the connection to ACS.

I'll add some comments accordingly.